src/pages/loginPage.tsx

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import React, {useState} from 'react';
+import {
+  View,
+  Text,
+  TextInput,
+  Button,
+  Alert,
+  ActivityIndicator,
+  KeyboardAvoidingView,
+  Platform,
+} from 'react-native';
+import {loginAccess} from '../service/client';
+import {Navigation} from 'react-native-navigation';
+import styles from '../styles/loginPageStyles';
+
+interface PageProps {
+  componentId: string;
+  rootTag: number;
+}
+
+const loginPage = (props: PageProps) => {
```

Comment:
A gente costuma nomear componentes React com a primeira letra maiúscula. Ficaria assim:
```suggestion
const LoginPage = (props: PageProps) => {
```